build.sbt

```diff
@@ -35,12 +35,11 @@ libraryDependencies ++= Seq(
   "com.github.scopt" %% "scopt" % "3.7.0",
   "com.sun.mail" % "javax.mail" % "1.6.2",
   "com.lihaoyi" %% "scalatags" % "0.6.7",
-  "io.circe" %% "circe-yaml" % "0.9.0",
+  "io.circe" %% "circe-yaml" % "0.10.1",
   "io.circe" %% "circe-core" % circeVersion,
```

Comment:
Should instead bump circeVersion variable to `0.10.1`.

Be careful there is likely a reason this was disjointed from the variable version.

Reply:
circe-yaml 0.10.x [depends on circe 0.11.1](https://github.com/circe/circe-yaml/blob/v0.10.1/build.sbt#L16).

circe-yaml 0.9.0 [depends on circe 0.10.0](https://github.com/circe/circe-yaml/blob/v0.9.0/build.sbt#L18).

They're disjointly released, frustratingly.

I think what I'll do is let this PR go through and let `sbt` decide which version of circe gets used. If the tests pass, great. circe 0.11.x has Scala 2.11 packages so we should be safe there.